### ISSUE TITLE
Common Navigation Needed for User Profiles to Solve Dead-Ends

### ISSUE THREAD

**Issue by alexbfree** [zooniverse/Panoptes-Front-End#2212]
I noticed that it is very awkward to navigate between your profile, collection, favourites, and stats, as many of the menu options disappear

When you first click "Profile", you get

![screenshot 2016-02-15 16 53 04](https://cloud.githubusercontent.com/assets/1473244/13055159/a615fb78-d405-11e5-9e60-bd84cc88d1fc.png)

If you then click Collections or Favourites you get:

![screenshot 2016-02-15 16 53 09](https://cloud.githubusercontent.com/assets/1473244/13055175/c1abf7b6-d405-11e5-80fe-0330fbbf1c56.png)

Notice the "Recent Comments" option has disappeared and is no longer accessible, nor is "Your stats"

If you then click into Collections, or Favourites, you get
![screenshot 2016-02-15 16 54 58](https://cloud.githubusercontent.com/assets/1473244/13055196/db751740-d405-11e5-8b23-a13807af69ba.png)
![screenshot 2016-02-15 16 53 26](https://cloud.githubusercontent.com/assets/1473244/13055195/db74738a-d405-11e5-91d7-d3d4588b3380.png)

Notice that there is no way to navigate to Recent Comments, or Your Stats, and no way to navigate to the opposite one of Collections/Favourites.

We should have a common navigation across all "Profile" pages. Right now you have to click back on your Profile and start all over again if you are in anything other than Recent Comments.

**Comment by alexbfree** [comment]
Further to this, the navigation when viewing another user's profile is also highly confusing.
For example when visiting 
https://www.zooniverse.org/users/gaga7

You are subject to all the problems above, but also, when you click Collections, you get to 
https://www.zooniverse.org/collections/gaga7 

which contains the same "submenu"
![screenshot 2016-02-16 14 59 21](https://cloud.githubusercontent.com/assets/1473244/13079807/e9d2d224-d4bd-11e5-818a-40176ee64aa8.png)

however none of these options relate to the user you are viewing, nor the current view. All includes all collections from everyone, and the other two links take you into your own collections and favourites.

**Comment by alexbfree** [comment]
Another related issue. If you click View collection from the All Collections view, not only do you lose the context of which user's collection it is (only user avatar remains) but you have no nav menu at all
e.g. https://www.zooniverse.org/collections/gardenmaeve/curious
![screenshot 2016-02-16 15 52 38](https://cloud.githubusercontent.com/assets/1473244/13081603/55c6806e-d4c5-11e5-8345-bb74b972635b.png)

**Comment by alexbfree** [comment]
We should also hide "Page [1] of 1" nav selector control when there is only one page.

**Comment by alexbfree** [comment]
It turns out my fix for #2253 that is under development will fix almost all these issues. Need to retest after #2253 is closed

**Comment by alexbfree** [comment]
#2253 addresses everything apart from https://github.com/zooniverse/Panoptes-Front-End/issues/2212#issuecomment-184742772 which we will do in a separate ticket as it's less critical.

**Comment by alexbfree** [comment]
Fixed by #2804 .
